packages/webpack-config/package.json

```diff
@@ -40,8 +40,8 @@
     "url": "https://github.com/expo/expo-cli/issues"
   },
   "dependencies": {
-    "@babel/core": "7.9.0",
-    "babel-loader": "8.1.0",
+    "@babel/core": "7.18.13",
+    "babel-loader": "^8.2.5",
```

Comment:
This is a critical change which effects how modules are bundled on web, needs extensive testing on a variety of projects to ensure Webpack bundling continues to work as expected.